the-collab-lab/tcl-26-smart-shopping-list
# Create hide and show logic for joining list on Homepage

## Description



This adds logic on Home.js to show and hide the form for joining an existing list when you click "Join List". I moved the join-list-form into its own component. 

If there is a better way of doing this, I'm open to suggestion! I can also use some help with getting the CSS just right.

I also made some small changes so that the Homepage is getting closer to the wireframes, mostly by changing copy and getting rid of the <header> since we will be putting an image next to the text, adding our app logo, etc.

## Related Issue



Closes #54 : Conditional logic showing list on Homepage

## Acceptance Criteria



N/A - I didn't have time to write any in the ticket when we created it!

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before



![image](https://user-images.githubusercontent.com/61371242/131204342-71c04e4d-0d16-4127-bfe9-3dca4deb7c57.png)


### After



![Aug-29-2021 23-31-41](https://user-images.githubusercontent.com/61371242/131281189-7cced06a-b29d-4361-b7b0-0fd8dbf41378.gif)



## Testing Steps / QA Criteria



1. Make sure you don't have a token so that you see the Homepage.
2. On the Homepage, you should see "Join List". 
2. When you click "Join List", the join list form appears with additional copy.
3. The button "changes" to "Join This List" instead of "Join List".
